Core/Scripts: simplify speech for DK races quests

TecDian opened the pull request:
**Changes proposed:**
-  eliminate the switch for the 10 races, use the same code for all
-  give the 10 npcs the correct texts with correct references to broadcast_text

**Target branch(es):** 3.3.5/master

- [X] 3.3.5

**Issues addressed:** Closes #19536


**Tests performed:** Built and tested in-game


**Known issues and TODO list:** (add/remove lines as needed)

Nothing that I know of.

TecDian committed:
simplify speech for quest dk race quest "A Special Surprise" (ID 12739,12742-12750)

tkrokli commented on src/server/scripts/EasternKingdoms/ScarletEnclave/chapter2.cpp:
```diff
@@ -729,310 +686,36 @@ class npc_a_special_surprise : public CreatureScript
-                                    me->SetHealth(0);
-                                    return;
-                            }
+                        case 0: Talk(SAY_EXEC_START, player); break;
```
TC codestyle: place `break;` on its own line below the `case` lines and indent it.

TecDian committed:
Update chapter2.cpp

joschiwald commented on sql/updates/world/3.3.5/2017_xx_xx_xx.sql:
```diff
@@ -0,0 +1,144 @@
+-- Malar Bravehorn, tauren
+DELETE FROM `creature_text` WHERE `entry`=29032;
+INSERT INTO `creature_text` (`entry`, `groupid`, `id`, `text`, `type`, `language`, `probability`, `emote`, `duration`, `sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES 
+(29032, 0, 0, (SELECT `MaleText` FROM `broadcast_text` WHERE `ID`=29286), 12, 0, 100, 0, 0, 0, 29286, 0, 'special_surprise SAY_EXEC_START'),
```
don't use subquerys with broadcast_text

DDuarte replied:
This should execute the subquery only once, what's the issue?

joschiwald replied:
cherry pick to master

Commit:
Update 2017_xx_xx_xx.sql

- replace SELECT sub-query with actual broadcast_text content
  to make the PR content compatible for cherry-pick to master

Commit:
Update 2017_xx_xx_xx.sql

- insert updated and correct column names for the creature_text table
- merge 11 queries into 1 query because the query syntax is identical

Aokromes committed:
Rename 2017_xx_xx_xx.sql to 2017_06_01_00_world.sql

ccrs commented on src/server/scripts/EasternKingdoms/ScarletEnclave/chapter2.cpp:
```diff
@@ -729,310 +686,49 @@ class npc_a_special_surprise : public CreatureScript
+                        case 11:
+                            Talk(EMOTE_DIES);
+                            me->setDeathState(JUST_DIED);
+                            me->SetHealth(0);
```
needed?

tkrokli replied:
That is the original script code, the only change is indentation.
If you have got a better suggestion, feel free to post it.

Aokromes committed:
Rename 2017_06_01_00_world.sql to 2017_06_05_00_world.sql

Aokromes committed:
Rename 2017_06_05_00_world.sql to 2017_06_05_01_world.sql

tkrokli commented:
Thank you for this contribution, @TecDian ! Nice to see the script simplified like this. 👍
